src/base/components/KAbout.vue

```diff
@@ -11,6 +11,22 @@
       </div>
 
       <QList>
+        <QItem
+          tag="a"
+          rel="noopener noreferrer"
+          href="https://community.foodsaving.world/t/how-to-get-involved-onboarding-into-the-karrot-team/661"
+          target="_blank"
+        >
+          <QItemSection side>
+            <QIcon name="fas fa-users" />
+          </QItemSection>
+          <QItemSection>
+            <QItemLabel>
+              How to Get Involved
```

Comment:
Text should be put into translation system.

Reply:
@brnsolikyl mentioned that it maybe makes sense to keep it in English, given the "team" is only communicating in English. Makes sense to me, but would also include a code-level comment to explain why it's not translated, e.g.:

```html
<!-- this is in English because -->
```